ENIGMAsystem/SHELL/Universal_System/Extensions/Json/json_reader.cpp

```diff
@@ -845,7 +847,9 @@ std::istream& operator>>( std::istream &sin, Value &root )
     Json::Reader reader;
     bool ok = reader.parse(sin, root, true);
     //JSON_ASSERT( ok );
+    #if !defined(JSON_READER_TEST_CPP)
     if (!ok) DEBUG_MESSAGE(reader.getFormattedErrorMessages(), MESSAGE_TYPE::M_ERROR);
+    #endif // JSON_READER_TEST_CPP
```

Comment:
This ``DEBUG_MESSAGE(reader.getFormattedErrorMessages(), MESSAGE_TYPE::M_ERROR);`` causes an error as ``enigma_user::show_debug_message()`` is not recognized. I think we need to include the source file which contains the implementation for ``enigma_user::show_debug_message()``. Anyway, we don't need it in testing so we remove it using header guards. We also remove ``#include "Widget_Systems/widgets_mandatory.h"``.